File: app/src/main/java/com/tendebit/dungeonmaster/charactercreation/classselection/model/CharacterClassManifest.java
package com.tendebit.dungeonmaster.charactercreation.classselection.model;

import com.google.gson.annotations.SerializedName;
import com.tendebit.dungeonmaster.charactercreation.classselection.model.CharacterClassDirectory;

import java.util.List;

public class CharacterClassManifest {

    @SerializedName("results")
    public List<CharacterClassDirectory> characterClassDirectories;
}

File: app/src/main/java/com/tendebit/dungeonmaster/charactercreation/pages/classselection/model/CharacterClassManifest.kt
package com.tendebit.dungeonmaster.charactercreation.pages.classselection.model

import com.google.gson.annotations.SerializedName

/**
 * Top-level list of [CharacterClassDirectory] as returned by the dnd5e API
 */
class CharacterClassManifest {
    @SerializedName("results")
    lateinit var characterClassDirectories: List<CharacterClassDirectory>
}